com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Configuration.cs

```diff
@@ -493,22 +510,36 @@ void FixDefaultVolumeProfileAssigned(bool fromAsyncUnused)
             if (!IsHdrpAssetUsedCorrect())
                 FixHdrpAssetUsed(fromAsync: false);
 
-            var hdrpAsset = HDRenderPipeline.currentAsset;
+            var hdrpAsset = HDRenderPipeline.defaultAsset;
```

Comment:
@jenniferd-unity  this will have an impact on your PR, rights?

Reply:
i already fixed that in my PR actually :) but i ll make sure i dont break it.